### PR TITLE
DeaDBeef: add version 2019-09-08

### PR DESCRIPTION
**Pull request by linsui**

### CODE REVIEW

File: bucket/deadbeef.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "version": "2019-09-08",
+    "description": "A music player",
```
Comment (Ash258) [inline bucket/deadbeef.json]: There is not mentioned it is unofficial port of this player for unix based OSes

File: bucket/deadbeef.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "homepage": "https://deadbeef-for-windows.github.io/",
```
Comment (Ash258) [inline bucket/deadbeef.json]:
```suggestion
    "homepage": "https://deadbeef-for-windows.github.io",
```

File: bucket/deadbeef.json (hunk 1 above)
Comment (linsui) [reply bucket/deadbeef.json]: What's your suggestion? How about `A port of DeaDBeeF music player.` ?

Comment (Ash258) [reply bucket/deadbeef.json]: > There is not mentioned it is unofficial port

http://deadbeef.sourceforge.net/download.html
> Windows builds
> DeaDBeeF builds for Windows
> Please note, that while it should be possible to build DeaDBeeF for Windows from the official git repository, the builds above are not official and not supported. Please report issues to the DeaDBeeF for Windows Project.

Comment (linsui) [reply bucket/deadbeef.json]: So `Unofficial port of DeaDBeeF music player.` ?

**Pull request merged**